core/src/test/java/io/opencensus/stats/ViewTest.java

```diff
@@ -13,118 +13,78 @@
-        Arrays.asList(Duration.fromMillis(1), Duration.fromMillis(22), Duration.fromMillis(333)));
-    final View view = IntervalView.create(
-        name, description, measure, iAggr, keys);
-
+    final View view = View.create(
+        name, description, measure, aggregations, keys, IntervalWindow.create(minute));
     assertThat(view.getName()).isEqualTo(name);
     assertThat(view.getDescription()).isEqualTo(description);
     assertThat(view.getMeasure().getName())
         .isEqualTo(measure.getName());
+    assertThat(view.getAggregations()).isEqualTo(aggregations);
     assertThat(view.getColumns()).hasSize(2);
     assertThat(view.getColumns().get(0).toString()).isEqualTo("foo");
     assertThat(view.getColumns().get(1).toString()).isEqualTo("bar");
-    assertTrue(view.match(
-        new Function<DistributionView, Boolean> () {
-          @Override public Boolean apply(DistributionView dView) {
-            return false;
-          }
-        },
-        new Function<IntervalView, Boolean> () {
-          @Override public Boolean apply(IntervalView iView) {
-            return iView == view;
-          }
-        }));
+    assertThat(view.getWindow()).isEqualTo(RpcViewConstants.INTERVAL_MINUTE);
   }
 
   @Test
   public void testViewEquals() {
-    DistributionAggregation dAggr = DistributionAggregation.create();
-    IntervalAggregation iAggr = IntervalAggregation.create(
-        Arrays.asList(Duration.fromMillis(1), Duration.fromMillis(22), Duration.fromMillis(333)));
     new EqualsTester()
         .addEqualityGroup(
-            DistributionView.create(
-                name, description, measure, dAggr, keys),
-            DistributionView.create(
-                name, description, measure, dAggr, keys))
+            View.create(
+                name, description, measure, aggregations, keys, CumulativeWindow.create()),
+            View.create(
+                name, description, measure, aggregations, keys, CumulativeWindow.create()))
         .addEqualityGroup(
-            DistributionView.create(
-                name, description + 2, measure, dAggr, keys))
+            View.create(
+                name, description + 2, measure, aggregations, keys, CumulativeWindow.create()))
         .addEqualityGroup(
-            IntervalView.create(
-                name, description, measure, iAggr, keys),
-            IntervalView.create(
-                name, description, measure, iAggr, keys))
+            View.create(
+                name, description, measure, aggregations, keys, IntervalWindow.create(minute)),
+            View.create(
+                name, description, measure, aggregations, keys, IntervalWindow.create(minute)))
         .addEqualityGroup(
-            IntervalView.create(
-                name, description + 2, measure, iAggr, keys))
+            View.create(
+                name, description + 2, measure, aggregations, keys, IntervalWindow.create(minute)))
```

Comment:
Suggestion: It might be better to modify the interval size instead of the description, since the equality group above already modifies the description.